src/frontend/src/styles/main.css

```diff
@@ -2158,6 +2180,80 @@ input[type="checkbox"] {
   image-rendering: pixelated;
 }
 
+/* Marquee */
+
+.c-marquee {
```

Comment:
Can you clarify how the marquee generally works? How are some rows going to the left and some to the right?

Reply:
Did so in 5de61a6196873d82d06a4d68ddd301835d660730